docs/debugging.md

```diff
@@ -0,0 +1,79 @@
+## Debugging
+
+Debugging GEF has a trick, let's see some examples
+
+## Debugging a command execution with pdb
+
+Open gef.py
+
+Search for **class NopCommand(GenericCommand)**, go to do_invoke method and insert:
+
+```python
+import pdb; pdb.set_trace()
```

Comment:
We support from Python 3.6 but it could be worth mentioning Python 3.7 has a builtin [`breakpoint()`](https://docs.python.org/3/library/pdb.html) function.

Reply:
So we support from 3.6 and if our users are using the min 3.6 supported version in the doc we are mentioning a dont available feature for them? I dont like it, dont makes sense from me